wordpress/wp-admin/css/code-editor-rtl.css

```
@@ -0,0 +1,77 @@
+	position: absolute;
+	top: 0;
+}
+
+.wrap [class*="CodeMirror-lint-marker"]:before {
+	font: normal 18px/1 dashicons;
+	position: relative;
+	top: -2px;
+}
+
+.wp-core-ui [class*="CodeMirror-lint-message"]:before {
+	font: normal 16px/1 dashicons;
```

Biome (2.1.2)

[error] Generic font family missing.

Consider adding a generic font family as a fallback.
For examples and more information, see the MDN Web Docs


(lint/a11y/useGenericFontNames)